excel-importer/src/test/java/uk/gov/hmcts/ccd/definition/store/excel/service/ImportServiceImplTest.java

```diff
@@ -280,7 +279,7 @@ public void shouldNotImportDefinition() throws Exception {
         service.importFormDefinitions(inputStream);
     }
 
-    @Test(expected = MissingUserRolesException.class)
+    @Test(expected = MissingAccessProfilesException.class)
     public void importDefinitionThrowsMissingUserRole() throws Exception {
```

Comment:
Rename test `importDefinitionThrowsMissingUserRole`

Reply:
Fixed in 6dc5c13